src/main/java/app/backend/controller/optimization/OptimizationController.java

```diff
@@ -146,13 +147,12 @@ public ResponseEntity<Void> orderOptimization(
 
         OptimizationRequest optimizationRequest;
         try {
-            optimizationRequest = optimizationUtils.getOptimizationRequest(crossroadId, startTimeId, optimizationTime);
+            optimizationRequest = optimizationUtils.getOptimizationRequest(crossroadId, startTimeId, optimizationTime, 3);
```

Comment:
Daj może to 3 jako jakaś zmienna typu BUS_SCALING na górze klasy czy coś

Reply:
to co innego akurat:P